src/llhttp/http.ts

```diff
@@ -313,8 +321,23 @@ export class HTTP {
     };
 
     // Response
+    const endResponseProtocol = () => {
+      return this.span.protocol.end(
+        this.invokePausable('on_protocol_complete', ERROR.CB_PROTOCOL_COMPLETE, n('res_after_protocol'),
+        ));
+    }
+
     n('start_res')
```

Comment:
this state looks odd since it only has an otherwise condition, would it be possible to merge it with ` n('start_res')`

Reply:
Unfortunately not. It is used to start tracking the span for the protocol and then we move to `res_after_start` to actually match the protocol.